security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,53 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-57"]
+== OpenShift Compliance Operator 0.1.57
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.57:
+
+* link:https://access.redhat.com/errata/RHBA-2022:6657[RHBA-2022:6657 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-57-new-features-and-enhancements"]
+=== New features and enhancements
+
+* `KubeletConfig` checks changed from `Node` to `Platform` type. `KubeletConfig` checks the default configuration of the `KubeletConfig`. The configuration files are aggregated from all nodes into a single location per node pool. See xref:../../security/compliance_operator/compliance-operator-remediation.adoc#compliance-evaluate-kubeletconfig-rules[Evaluating `KubeletConfig` rules against default configuration values].
+
+* The `ScanSetting` Custom Resource now allows users to override the default CPU and memory limits of scanner pods through the `scanLimits` attribute.  For more information, see xref:../../security/compliance_operator/compliance-operator-troubleshooting.adoc#compliance-increasing-operator-limits[Increasing Compliance Operator resource limits].
+
+* A `PriorityClass` object can now be set through `ScanSetting`. This will ensure the Compliance Operator is prioritized and minimizes the chance that the cluster will fall out of compliance. For more information, see xref:../../security/compliance_operator/compliance-operator-advanced.adoc#compliance-priorityclass_compliance-advanced[Setting `PriorityClass` for `ScanSetting` scans].
+
+[id="compliance-operator-0-1-57-bug-fixes"]
+=== Bug fixes
```

Comment:
For the bug fixes, it only highlight the bugs from bugzilla. Do we need to add the bugs from Jira issues? Thanks.

Reply:
@rhmdnd can you confirm if any of the Jira issues need to be added to the bug fix section of this release? It seems like [OCPBUGS-2331](https://issues.redhat.com/browse/OCPBUGS-2331) and [OCPBUGS-1066](https://issues.redhat.com/browse/OCPBUGS-1066) are bug fixes since the last downstream release, correct? Can you assist with the cause/consequence/fix/result in the Jiras so I can update the bug fixes for this release?

Reply:
We may have to address additional bugs post-release, as this content is now live and needs to be merged. The errata points to this documentation.